src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -234,6 +234,18 @@ inline float avx2_load_f32(const int8_t* data, ov::element::Type type) {
     }
 }
 
+inline __m256 u4_avx2_load_f32(const uint8_t* data) {
```

Comment:
kind of name for loading low u4 parts only not good. 
avx2_load_f32 just with type u4, or u4 + sign low/high part would be enough